<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waitPost</class>
 <widget class="QWidget" name="waitPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>等待报名开始</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>当前时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="current_time">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>2022年1月2日</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>开始时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="start_time">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>2022年1月2日</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>离开报名时间还有：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>22分32秒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
